src/test/java/competition/subsystems/turret/TurnCommandsTest.java

```diff
@@ -0,0 +1,44 @@
+package competition.subsystems.turret;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import competition.BaseCompetitionTest;
+import competition.subsystems.turret.commands.TurnLeft;
+import competition.subsystems.turret.commands.TurnRight;
+import competition.subsystems.turret.commands.TurretStop;
+
+public class TurnCommandsTest extends BaseCompetitionTest {
+
+    @Test
+    public void testTurnLeft() {
+        TurretSubsystem turret = this.injector.getInstance(TurretSubsystem.class);
+
+        TurnLeft turnLeft = new TurnLeft(turret);
+        turnLeft.initialize();
+        turnLeft.execute();
+        assertEquals(0.03 , turret.motor.getMotorOutputPercent(), 0.001);
+    }
+
+    @Test
+    public void testTurnRight() {
+        TurretSubsystem turret = this.injector.getInstance(TurretSubsystem.class);
+        TurnRight turnRight = new TurnRight(turret);
+        turnRight.initialize();
+        turnRight.execute();
+
+        assertEquals(-0.03 , turret.motor.getMotorOutputPercent(), 0.001);
+    }
+
+    @Test
+    public void testStop() {
+        TurretSubsystem turret = this.injector.getInstance(TurretSubsystem.class);
+        TurretStop stopTurret = new TurretStop(turret);
+        stopTurret.initialize();
+        stopTurret.execute();
+
```

Comment:
This doesn't actually test if the turret stops, because it was never moving to begin with.